golem/managers/proposal/plugins/scoring/scoring_buffer.py

```diff
@@ -1,33 +1,26 @@
 import asyncio
 import logging
 from datetime import timedelta
-from typing import List, Optional
+from typing import Optional
 
-from golem.managers.proposal.plugins.buffer import Buffer
-from golem.managers.proposal.plugins.scoring.mixins import ProposalScoringMixin
+from golem.managers.proposal.plugins.buffer import Buffer as BufferPlugin
```

Comment:
Maybe we should rename `class Buffer(ProposalManagerPlugin):` to `BufferPlugin` now that we have `BackgroundFillBuffer` and others?